### PR TITLE
roma xyi

### PR DESCRIPTION
**Pull request by AndriiSolovei**

### CODE REVIEW

**Comment by AndriiSolovei** [comment]
соси дура

File: tic_tac-toe.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        result.push([newArr[i], newArr[i+1], newArr[i+2]].join("|"));
+    }
+    console.log(result);
+return result.join("\n---+---+---\n");    
+}
+console.log(renderField([
+    1,0,0,
+    0,0,0,
+    0,0,-1
+]));
```
Comment (romashmigol) [inline tic_tac-toe.js]: Add EOL

File: tic_tac-toe.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+function renderField (field_2) {
+    let newArr = changeValue(field_2);
+    let result = [];
+    for (i = 0; i < newArr.length; i+=3) {
+        result.push([newArr[i], newArr[i+1], newArr[i+2]].join("|"));
+    }
+    console.log(result);
```
Comment (romashmigol) [inline tic_tac-toe.js]: Delete the console.log

File: tic_tac-toe.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            newArr = field.map(el => { 
+                if (el == 1) {
+                    return ' x ';
+                } else if (el == -1) {
+                    return ' 0 ';
+                } else {
+                    return '   ';
+                }
+            });
+return newArr;
```
Comment (romashmigol) [inline tic_tac-toe.js]: Add 4 spaces here

File: tic_tac-toe.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+function renderField (field_2) {
+    let newArr = changeValue(field_2);
+    let result = [];
+    for (i = 0; i < newArr.length; i+=3) {
+        result.push([newArr[i], newArr[i+1], newArr[i+2]].join("|"));
+    }
+    console.log(result);
+return result.join("\n---+---+---\n");
```
Comment (romashmigol) [inline tic_tac-toe.js]: Add 4 spaces here

File: tic_tac-toe.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+function changeValue (field) {
+    let newArr;
```
Comment (romashmigol) [inline tic_tac-toe.js]: Why did you declare variable here?

File: tic_tac-toe.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+            });
+return newArr;        
+}
+
+
+
+function renderField (field_2) {
+    let newArr = changeValue(field_2);
+    let result = [];
+    for (i = 0; i < newArr.length; i+=3) {
```
Comment (romashmigol) [inline tic_tac-toe.js]: Add spaces around the '+='

**Review by romashmigol — CHANGES_REQUESTED** [review]

**Review by romashmigol — APPROVED** [review]
The cost of this task is **25 coins** in the market day

**Pull request merged**